compiler-core/src/language_server/engine.rs

```diff
@@ -175,6 +175,11 @@ where
                 None => return Ok(None),
             };
 
+            let a =
+                node.definition_location(this.compiler.project_compiler.get_importable_modules());
+            tracing::info!("{:#?}", node);
+            tracing::info!("{:#?}", a);
```

Comment:
Debugging left in here.

Use the debug API `dbg!` for debugging rather than the logging API `tracing` as then then CI will let you know if you left it in.

Reply:
Ah I had no idea 😆 thanks for the heads up!